Bundle/FilterBundle/Domain/FilterFormFieldQueryHandler.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Victoire\Bundle\FilterBundle\Domain;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\EntityManager;
+use Victoire\Bundle\QueryBundle\Helper\QueryHelper;
+use Victoire\Bundle\WidgetBundle\Entity\Widget;
+
+/**
+ * Class FilterFormFieldQueryHandler.
+ */
+class FilterFormFieldQueryHandler
+{
+    /**
+     * @var QueryHelper
+     */
+    private $queryHelper;
+    /**
+     * @var EntityManager
+     */
+    private $entityManager;
+
+    /**
+     * FilterFormFieldQueryHandler constructor.
+     *
+     * @param QueryHelper   $queryHelper
+     * @param EntityManager $entityManager
+     */
+    public function __construct(
+        QueryHelper $queryHelper,
+        EntityManager $entityManager
+    ) {
+        $this->queryHelper = $queryHelper;
+        $this->entityManager = $entityManager;
+    }
+
+    public function handle(Widget $widgetFilter, $entity)
```

Comment:
add some documentation (at least phpdoc) to explane what is the function's purpose